core/block_render_svg_vertical.js

```diff
@@ -1146,7 +1156,7 @@ Blockly.BlockSvg.prototype.renderDrawTop_ = function(steps, rightEdge) {
   /* eslint-disable indent */
   if (this.type == Blockly.BlockSvg.DEFINE_BLOCK_TYPE) {
     steps.push('m 0, 0');
-    steps.push('v', -2 * Blockly.BlockSvg.GRID_UNIT);
+    //steps.push('v', -2 * Blockly.BlockSvg.GRID_UNIT);
```

Comment:
Should this line be removed instead of commented out?

Reply:
Good catch.  Done.